Release/include/pplx/pplxtasks.h

```diff
@@ -1460,7 +1464,7 @@ namespace details
         virtual ~_ContinuationTaskHandleBase() {}
     };
 
-#if _PPLTASK_ASYNC_LOGGING
+#if _PPLTASK_ASYNC_LOGGING && _MSC_VER >= 1800 && defined(__cplusplus_winrt)
```

Comment:
Hmm it isn't clear why this test isn't merged into _PPLTASK_ASYNC_LOGGING as it was before?